src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -280,6 +254,33 @@ function collapse(panels) {
     }
 }
 
+function toggleCollapse(e, pans) {
+    var EXPAND = 'Expand';
+    var COLLAPSE = 'Collapse';
+    var panels = pans || $('div.panel-collapse');
+    var isElementAnExpandButton = $(e).html().trim().startsWith(EXPAND);
+
+    if (isElementAnExpandButton) {
+        var isElementCollapsePanelsButton = $(e).is($('#collapse-panels-button'));
+        // We might need to load {@code panels} data by Ajax if {@link #toggleCollapse} is invoked by
+        // clicking on collapse panels button.
+        var mightNeedAjaxLoading = isElementCollapsePanelsButton;
+        expand(panels, mightNeedAjaxLoading);
+        var htmlString = $(e).html();
+        htmlString = htmlString.replace(EXPAND, COLLAPSE);
```

Comment:
wow didnt think of that and I think it is a good suggestion. Just made the change and I used replaceHTMLString as the method name. Any comment on that?